shared/base/view.js

```diff
@@ -195,7 +195,7 @@ module.exports = BaseView = Backbone.View.extend({
    * Get the HTML for the view, including the wrapper element.
    */
   getHtml: function() {
-    var html = this.getInnerHtml(),
+    var html = this.options.lazy ? '' : this.getInnerHtml(),
```

Comment:
sorry, wrapper div != placeholder data. i more mean like if you have something loading and you just want to display a placeholder image and some default text, then after you're done fetching you can replace that fake information with the data from the API. With this change it will be impossible to do that, whereas fixing the place it's invoking this function will be a better choice.

it'd probably be better to add a function and allow an override to chose if you just want the wrapper div or a version of the template w/ fake data (how it's working now)